src/Components/WebAssembly/WebAssembly/src/Hosting/WebAssemblyHostBuilder.cs

```diff
@@ -59,6 +62,37 @@ private WebAssemblyHostBuilder()
             {
                 return Services.BuildServiceProvider();
             };
+
+            InitializeEnvironment();
+        }
+
+        private void InitializeEnvironment()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Create("WEBASSEMBLY")))
```

Comment:
Could you update the comment to clarify this is for the benefit of unit tests? At first I was a bit confused about why we were checking this here given that the class name strongly implies this is only intended to be used on WebAssembly 😄 

Maybe even move the guard clause to surround L66, though up to you of course!

Reply:
I clarified that this is test only